docs/ops/movement/Unique_10.md

```diff
@@ -0,0 +1,111 @@
+# Unique  {#openvino_docs_ops_comparison_Unique_10}
```

Comment:
```suggestion
# Unique  {#openvino_docs_ops_movement_Unique_10}
```

Reply:
This line should be aligned with the directory of the op category. Actually in this case rather the `Unique_10.md` should be moved from `/movement` to `/comparison` directory, instead of changing the `{#openvino_docs_ops_comparison_Unique_10}`.